Remotion/Web/Core/Schemas/WebConfiguration.xsd

```diff
@@ -4,7 +4,8 @@
     xmlns:c="http://www.re-motion.org/web/configuration" 
     elementFormDefault="qualified" 
     attributeFormDefault="unqualified" 
-    targetNamespace="http://www.re-motion.org/web/configuration">
+    targetNamespace="http://www.re-motion.org/web/configuration"
+    version="2.0">
```

Comment:
I don't know about this. We usually did 
```suggestion
    targetNamespace="http://www.re-motion.org/web/configuration/v2">
```

which does require updating the config files

Reply:
Thanks. And yeah...

Reply:
Just because I saw it with the WXE Url mapping XML schema: There we are using `/1.0` as suffix. Do you want to leave it here with `/v2` or use `/2.0` to keep it in sync?

Reply:
It's "as you like."

The major.minor syntax was meant to convey if there's compatible or incompatible changes to the config. Since Namespace is always a breaking change, the major.minor syntx is kind of useless. It might be easier to read, though.